js-inputs-and-strings/js-inputs-and-strings_switch-values/js/index.js

```diff
@@ -19,8 +19,29 @@ const lowercaseButton = document.querySelector('[data-js="button-lowercase"]');
 
 uppercaseButton.addEventListener("click", () => {
   firstInput.value = firstInput.value.toUpperCase();
+  secondInput.value = secondInput.value.toUpperCase();
 });
 
 lowercaseButton.addEventListener("click", () => {
   firstInput.value = firstInput.value.toLowerCase();
+  secondInput.value = secondInput.value.toLowerCase();
+});
+
+const secondInput = document.createElement("input");
+secondInput.type = "text";
+secondInput.setAttribute("data-js", "second-input");
+document.querySelector("form").appendChild(secondInput);
+
+const switchButton = document.createElement("button");
+switchButton.type = "button";
+switchButton.innerHTML = "&updownarrow;";
+switchButton.setAttribute("data-js", "button-switch");
+document.querySelector("form").appendChild(switchButton);
+
+switchButton.addEventListener("click", () => {
+  const firstInputValue = firstInput.value;
```

Comment:
Not necessary to create new variables here and in the line below. You can directly use firstInput.value whereever you need it.